### ISSUE TITLE
The Training Details when you reproduce the PixelNeRF on DTU

### ISSUE THREAD

**Issue by Peggy-Y** [LRLVEC/regnerf#1]
I wonder the epochs or iterations you used for reproducing the pixelNeRF on DTU, cause I found that the results for 3-view evaluation are pretty different between yours, in PixelNeRF， they presented results of 19.33 PSNR and 0.695 SSIM, but in your paper, it is stated as 18.74 and 0.618.

**Comment by lateroo** [comment]
Hello, I would like to ask, in the color regularization part of the article, how is the loss function designed, and how to use the normalized flow model

**Comment by lateroo** [comment]
> 我想知道您用于在 DTU 上重现 pixelNeRF 的纪元或迭代，因为我发现 3 视图评估的结果与您的结果大不相同，在 PixelNeRF 中，它们提供了 19.33 PSNR 和 0.695 SSIM 的结果，但在您的论文中，它表示为 18.74 和 0.618。

Hello, I would like to ask, in the color regularization part of the article, how is the loss function designed, and how is the normalized flow model used? What do you think?
